cylc/flow/scheduler.py

```diff
@@ -1734,6 +1735,8 @@ async def main_loop(self) -> None:
                     self.pool.queue_task(itask)
 
             if housekeep_xtriggers:
+                # Spawn parentless clock-triggered
+                self.pool.spawn_parentless_clock_triggered()
                 # (Could do this periodically?)
                 self.xtrigger_mgr.housekeep(self.pool.get_tasks())
```

Comment:
Is this the right place to handle the spawning? The reason for the `if` block here is to "delete satisfied xtriggers no longer needed by any tasks" (from the `xtrigger_mgr.housekeep()` docstring).

Reply:
I noticed that `housekeep_xtriggers` is satisfied if xtriggers are, so just piggy backed off that.. But could just use that spawns list.